src/load_manager.cc

```diff
@@ -213,8 +213,6 @@ LoadManager::InitManagerInputs(
     const size_t string_length, const std::string& string_data,
     const bool zero_input, std::vector<std::string>& user_data)
 {
-  RETURN_IF_ERROR(factory_->CreateClientBackend(&backend_));
```

Comment:
This code had no downstream effects--this `backend_` wasn't used anywhere--random cleanup.

Reply:
where is the backend created if not here?

Reply:
There are several places a backend is created and later used. You can search for calls to `CreateClientBackend()` in the repository.

The one you're probably thinking of where the backend is used to send inference requests is here:

https://github.com/triton-inference-server/perf_analyzer/blob/a3e3d27895fab652e4d766e4fc86dd96a7723d6a/src/infer_context.h#L93